packages/@lwc/engine-dom/src/renderer/index.ts

```diff
@@ -193,6 +193,10 @@ function getTagName(elm: Element): string {
     return elm.tagName;
 }
 
+function attachInternals(elm: HTMLElement): ElementInternals {
+    return HTMLElement.prototype.attachInternals.call(elm);
```

Comment:
```suggestion
const attachInternalsFunc = HTMLElement.prototype.attachInternals;
function attachInternals(elm: HTMLElement): ElementInternals {
    return attachInternalsFunc.call(elm);
```

I'd feel safer about caching it from the prototype, just in case somebody tries to monkey-patch it later somehow.

Reply:
@nolanlawson I was thinking about this, I've cached the `attachInternals` method as suggested, however, I believe we do want to allow monkey-patching when the method needs to be polyfilled.

Is the intent to load all polyfills prior to loading LWC and prevent further monkey-patching while the engine is running?